lib/router/dynamic.ex

```diff
@@ -8,8 +8,19 @@ defmodule Router.Dynamic do
 
   plug(:match)
   plug(:fetch_query_params)
+  plug(:sanitize_site_param)
```

Comment:
this guy makes it so that our Elixir server only knows deals with valid site names after we get through this HTTP routing layer